### DISCUSSION TITLE
Irrigation in NoahMP

### DISCUSSION THREAD

**Discussion by ZhaoY-github** [NCAR/noahmp#134] [Q&A] [answered]
Hi there, 

i knew that there are three types of irrigation in the system as of now, sprinkler, dripping and tile drainage. I am wondering is there a spatial map of irrigation method that is already available, or we have to assume just one irrigation method when running the model? Thanks!

**Comment by cenlinhe** [comment, marked as answer]
yes, the irrigation map is already implemented into the WRF/WPS system. You could run WPS to generate geo_em file with irrigation variables. The irrigation total area map is global 1-km. The irrigation method fraction map is only for CONUS.

**Comment by ZhaoY-github** [comment]
Hi @cenlinhe, I found the relevant irrigation files in the geog/crop folder. Thanks for answering the question! Appreciated.
